README.md

```diff
@@ -62,12 +62,22 @@ Must be a list of *tensor specification keys*.
   *tensor specification keys*:
   - `name` tensor name
   - `data_type` data type (e.g. float32)
-  - `data_range` tuple of (minimum, maximum)
-  - `axes` string of axes identifying characters from: btczyx
+  - `data_range` tuple of (minimum, maximum) (e.g. [-inf, inf])
```

Comment:
Just a hunch: Did you remove the inf example, because you expect `data_range` to hold the actual minimum, maximum of the tensor?
I think for now it was intended as the threoretical data_range (that might be given by the `data_type`), which the example clarifies. (a comment might be clearer, though). In combination with the data type this is of limited use, but the actual (minimum,maximum) would confusing due to the batch dimension, which is arbitrary and thus has a random influence on these values. The actual, global data range might not be required, thus unnecessary to be computed, or not sufficient (e.g. for percentile normalization). I suggest not to change anything further for 0.3.0, and keep it as theoretical data range.

Reply:
I think putting in the "data range so that the model yields sensible results" makes sense for the data range:
https://github.com/bioimage-io/configuration/issues/55#issuecomment-735646024